src/App.test.tsx

```diff
@@ -92,8 +92,10 @@ describe('App component', () => {
       });
 
       const result = screen.queryByText(/Result/);
-
       expect(result).toBeInTheDocument();
+
+      const referenceLink = screen.getByRole('link', { name: 'Reference' });
```

Comment:
From what I can see is that this test is obsolete because the coverage is in `Result.test.tsx`

Reply:
I agree, not really needed to have this additionally.